web_src/js/features/repo-issue.js

```diff
@@ -691,5 +691,7 @@ export function initArchivedLabelHandler() {
   if (!document.querySelector('.archived-label-hint')) return;
   for (const label of document.querySelectorAll('[data-is-archived]')) {
     toggleElem(label, label.classList.contains('checked'));
+    // also toggle the divider
+    toggleElem($(label).next('.exclusive-scope'), label.classList.contains('checked'));
```

Comment:
See above

Reply:
Will test it rigorously.

Reply:
The usecase I imagine to be faulty is `two (or more) labels in scope, one archived, the other not`.

Reply:
The same I was thinking, with org labels as well.

Reply:
https://github.com/go-gitea/gitea/assets/80308335/551b274c-a613-4276-b2a8-0bc352fc1e5b
This is the current condition with org and repo labels with all possible usecase, 
One is failing, when we have archived label with scoped and normal label [with archived marked].